### ISSUE TITLE
Add Permissive to License Categories reported for Adds or Changes

### ISSUE THREAD

**Issue by mjherzog** [nexB/deltacode#105]
We need to report all License Categories for the factors including "Permissive" which appears to be the only missing.

**Comment by johnmhoran** [comment]
@mjherzog  `Public Domain` is also missing.

**Comment by mjherzog** [comment]
Somehow we should cover all license categories. Since these may/will change over time.

**Comment by johnmhoran** [comment]
Agreed @mjherzog .  I'm including both `Permissive` and `Public Domain` in the refactoring I'm currently working on in this issue.

I've listed below the eight categories DeltaCode currently addresses.  Is this an up-to-date list of the categories that could be present in the output of a ScanCode scan?
```
Commercial
Copyleft
Copyleft Limited
Free Restricted
Patent License
Permissive
Proprietary Free
Public Domain
```

**Comment by johnmhoran** [comment]
@majurg I have not (yet) made changes to the JSON input files for `test_json2csv.py` to conform to the newly-added license category content generated by DeltaCode.  I can do so once we have agreement on the underlying changes, although the JSON input changes don't appear to be necessary for the performance of this set of tests.

**Comment by steven-esser** [comment]
Yea, we probably dont need to do anything with those tests for now. 

We really dont need to touch those unless we make sweeping changes to our json or csv structure.

**Comment by steven-esser** [comment]
merged #108, closing.
